fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -814,6 +814,15 @@ public class Config extends ConfigBase {
     @ConfField
     public static int load_checker_interval_second = 5;
 
+    @ConfField(mutable = true)
+    public static long lock_checker_interval_second = 30;
+
+    /**
+     * Check of deadlock is time consuming. Open it only when tracking problems.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_deadlock_check = false;
```

Comment:
```suggestion
    public static boolean lock_checker_enable_deadlock_detection = false;
```

better to use same prefix for consistent features.

Reply:
done